build.gradle

```diff
@@ -26,6 +26,9 @@ ext {
     localGitCommitShort = grgit.head().getAbbreviatedId(7)
     localGitDirty = !grgit.status().clean
 
+    // sets the minimum launcher version that is output for the bootstrapper
+    launcherVersion = '2.0.2'
```

Comment:
https://github.com/runelite-extended/launcher/blob/master/build.gradle#L14

Bump for new launcher while we're at it?
New launchers are the first to support version enforcement anyways